cogs/weightcog.py

```diff
@@ -5,21 +5,33 @@
 import matplotlib.pyplot as plt
 import numpy as np
 from datetime import datetime, timedelta, date
-
+from typing import Dict, List, Final, Any
+from pathlib import Path
+import yaml
 
 def has_bot_input_perms(ctx):
     role = discord.utils.get(ctx.guild.roles, name="bot-input")
     return role in ctx.author.roles
 
 class WeightCog(commands.Cog):
-    def __init__(self, bot):
+    MOVING_AVG_PERIODS : Final[Dict[str, int]] = {
+        "weekly_avg": 7, "last_week": 7,
+        "monthly_avg": 30, "last_month": 30,
+        "yearly_avg": 365, "last_year": 365
+    }
+    # Header row of a user weight csv file
+    HEADER_ROW : Final[List[str]] = ["Date", "Weight"] 
+    
+    def __init__(self, bot : commands.Bot):
         self.bot = bot
-        self.data_folder = "./BSF-bot-data/weightcog/"
+        self.CONFIG_PATH: Final[str] = Path("./BOT_CONFIG.yaml")
```

Comment:
update to config.yaml

Reply:
I've updated it but the current "Files changed" don't include the config.